aiohttp/multipart.py

```diff
@@ -854,6 +856,8 @@ def _apply_content_transfer_encoding(self, stream):
         elif encoding == 'quoted-printable':
             for chunk in stream:
                 yield binascii.b2a_qp(chunk)
+        elif encoding == 'binary':
+            yield from stream
```

Comment:
Does not work for me (ClientRequestError: Can not write request body'), we need:
```
 for chunk in stream:
      yield chunk
```